docs/decisions/0002-files/Highways.csv

```diff
@@ -1,5 +1,5 @@
-,OpenLayers,OpenLayers+MB,OpenLayers+React,Leaflet,Mapbox,Mapbox React

```

Comment:
My local git converted the line-endings on all of these; I think perhaps because they were otherwise Windows-based line endings? I'm not sure. This issue seems related based on the console messages I was getting. https://stackoverflow.com/questions/5834014/lf-will-be-replaced-by-crlf-in-git-what-is-that-and-is-it-important

Reply:
Here's[ a good solution for this issue](https://stackoverflow.com/a/5834094).